build.sbt

```diff
@@ -106,11 +108,11 @@ scalacOptions ++= allVersionsOption ++ {
 }
 
 
-val ZioVersion = "2.0.0"
+val ZioVersion = "2.0.2"
 val ZioCatsVersion = "3.3.0"
 val DoobieVersion = "1.0.0-RC2"
-val AnormVersion = "2.6.10"
-val H2Version = "1.4.200"
+val AnormVersion = "2.7.0"
```

Comment:
Scala3 support recently added to anorm! 
https://github.com/playframework/anorm/pull/474
https://github.com/playframework/anorm/releases/tag/2.7.0